osde2e/managed_node_metadata_operator_tests.go

```diff
@@ -86,7 +100,6 @@ var _ = ginkgo.Describe("managed-node-metadata-operator", ginkgo.Ordered, func()
 			machineSetName = machineSet.GetName()
 			return machineSet.Status.ReadyReplicas == *machineSet.Spec.Replicas, nil
 		}, wait.WithTimeout(600*time.Second))
-		Expect(err).Should(BeNil(), "wait.For machinepool ready failed")
```

Comment:
I don't think we need to delete this one, we want to make sure that the wait returns a nil err

Reply:
Ah nice catch, I did move that into the big loop but must have forgotten to readd it, when that was moved to a smaller loop.